### PR TITLE
completed javascript-obeject exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

### CODE REVIEW

File: javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+var vehicle = {
+  make: 'Chevy',
+  model: 'Impala',
+  year: 1967
+};
+console.log('value of vehicle:', vehicle);
+
+vehicle['color'] = 'black';
+console.log('value of color:', vehicle['color']);
```
Comment (Zhiwutian) [inline javascript-objects/main.js:25]: you are missing the `isConvertible` property. make sure to add that!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

File: javascript-objects/main.js (hunk 1 above)
Comment (avallonsmith) [reply javascript-objects/main.js:25]: image

Whoops, sorry! It has been added!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**